backend/app.js

```diff
@@ -5,12 +5,10 @@ const debug = require('debug')('app');
 const routes = require('./routes/routes');
 const app = express();
 
-const config = require('./config.json');
-const port = config.port || 3000;
```

Comment:
Ik vond het niet verkeerd om de port configureerbaar te hebben hoor.

Reply:
dat is wel beter ja, netjes!